deep_quoridor/src/renderers/pygame.py

```diff
@@ -1,4 +1,3 @@
-import re
```

Comment:
I Removed the lines here just because the linter was complaining about them being unused.